build/test.ps1

```diff
@@ -38,7 +38,17 @@ function Test-One {
     }
 }
 
+function Test-VSCode {
+    Param([string] $path);
+
+    Write-Host "##[info]Testing VSCode Extension inside $path..."    
+    Push-Location (Join-Path $PSScriptRoot $path)
+        npm run unittest
```

Comment:
I think this should fix the break in the test pipeline.
```suggestion
        npm test
```

Reply:
I think this is failing because I forgot to edit the template `package.json`.

Reply:
Ah, I see the corresponding change below that adds `unittest` as one of the targets.